phoenix-core/src/main/java/org/apache/phoenix/monitoring/connqueryservice/ConnectionQueryServiceMetricsHistograms.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.phoenix.monitoring.connqueryservice;
```

Comment:
just thinking loud - can this package be org.apache.phoenix.monitoring.queryservice; or org.apache.phoenix.monitoring.connectionqueryservice (full name in package)?

Reply:
How are the metric classes in this package different from the ones in the monitoring package?

Reply:
This metric class is for CQS Metrics. We have similar metrics class for Global, Table etc.

Some example : 

`TableClientMetrics.java`
`GlobalClientMetrics.java`

This could be combined, I guess. I will add note in [PHOENIX-6174](https://issues.apache.org/jira/browse/PHOENIX-6174) to simplify in future in monitoring.